_posts/2019-06-07-python-datascience-with-openfaas.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Serving Data Science Models with OpenFaaS
+description: "Lucas will build and deploy a name classification function using OpenFaaS and PyTorch"
+date: 2019-06-01
+image: /images/pydatascience-template/artificial-neural-network-3501528_640.png
+categories:
+  - python
+  - coding
+  - examples
+  - datascience
+  - templates
+author_staff_member: lucas
+dark_background: false
+---
+
+Last month the OpenFaaS team was at [Kubecon Barcelona][kubecon-homepage], a frequent topic was deploying data science models to Kubernetes and, of course, can OpenFaaS help deploy models? In this post we will introduce a new function template aimed at Python data scientists and walk through a concrete example of deploying a [PyTorch][pytorch-homepage] model.
```

Comment:
Another way to say? ` a frequent topic was deploying data science models to Kubernetes` -> I / we were asked about. The topic came up?